src/Form/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { mapUrlParamsToHiddenFields } from '../utils/browser';
+import Client from '../utils/client';
+
+export function saveUrlParams(
```

Comment:
this utils file seems to serve the same purpose as formHelperFunctions, should probably be merged into one file